### PR TITLE
Updated UG for Reimbursement and Cashier 

### PR DESCRIPTION
**Pull request by dawo5010**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -509,17 +565,19 @@ You are advised to key in a number smaller than 10000 due to size limitation.
...
 `add [c/CATEGORY] d/DESCRIPTION q/QUANTITY`
 
-The quantity of the item inputted must not be greater than the available stock in the inventory. Else, Leo will display
-a message prompting input of a smaller quantity or another item. +
+The quantity of the item must not be greater than the stock available in the Inventory Tab.
+Else, Leo will display a message prompting input of a smaller quantity or another item.
+
 The category field is optional. If you are unsure about the description of the desired item, you can refer to
-the Inventory tab or simply key in the category without any other fields. Leo will display all the items in the
+the Inventory tab or simply key in the category without any other fields. +
```
Comment (EvonDong) [inline docs/UserGuide.adoc]: I think you missed this line. "t" of "tab" should be in capital letters too right?

Comment (dawo5010) [reply docs/UserGuide.adoc]: Yes, thank you!

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -107,6 +107,11 @@ after a command has been input
 it adds money to the club's funds).
 * The amount input should be negative if the transaction is an expenditure made by the club (that is, it deducts money
 from the club's funds).
+
+[WARNING]
+Do not key in a quantity where the total amount in table or subtotal of that row exceeds 2,147,483,647, the range
+of an integer. If it exceeds this number, please key in a smaller quantity or checkout items separately. +
+You are advised to key in a number smaller than 10000 due to size limitation.
```
Comment (EvonDong) [inline docs/UserGuide.adoc]: Actually, I estimated 10000. Maybe you can try to key in different inputs on the actual app to see how much space it takes up?

Comment (dawo5010) [reply docs/UserGuide.adoc]: I think it's okay, just leave it as a ballpark figure. We can KIV this for next week?

Comment (EvonDong) [reply docs/UserGuide.adoc]: Also, since decimals are allowed in our amounts, maybe you can state "digits" instead of an integer?

Comment (EvonDong) [reply docs/UserGuide.adoc]: Yup sure, there is no need to rush!

**Pull request merged**